server/poll/poll.go

```diff
@@ -32,11 +33,17 @@ type Settings struct {
 	PublicAddOption bool
 }
 
-// VotedAnswerResponse stores answers that is created by a user
-type VotedAnswerResponse struct {
-	PollID       string   `json:"poll_id"`
-	UserID       string   `json:"user_id"`
-	VotedAnswers []string `json:"voted_answers"`
+// Metadata stores personaized metadata of a poll.
+//
+// PollID is ID for poll
+// UserID is ID for user
+// AdminPermission will be true if the user with "UserID" has admin permission for the poll with "PollID". Otherwise AdminPermission will be false.
+// VotedAnswers is list of answer that the user with "UserID" have voted for the poll with "PollID"
+type Metadata struct {
```

Comment:
0/5 maybe move this and the other related code to a new file `server/poll/metadata.go`